components/browser/errorpages/src/main/java/mozilla/components/browser/errorpages/ErrorPages.kt

```diff
@@ -103,6 +103,10 @@ object ErrorPages {
             "&badCertTechInfo=$badCertTechInfo" +
             "&badCertGoBack=$badCertGoBack" +
             "&badCertAcceptTemporary=$badCertAcceptTemporary"
+
+        urlEncodedErrorPage = urlEncodedErrorPage
+            .replace("<ul>", "<ul role=\"presentation\">")
+        return urlEncodedErrorPage
```

Comment:
Hmmmm, those `<ul>` tags are comming from the translations?

Reply:
Yes. They are in all string.xml translation, they are used to show the items as a "list" with bullets.

Reply:
I'm not fully sure if we should land: Of course this works, but it also feels hacky! I wonder what @Delphine or @flodolo think here? Maybe it is finally time to break this HTML snippet up into multiple strings?

Reply:
Leaving the question (and product) back in Delphine's capable hands, since she's back ;-)

Reply:
So, I'm not entirely sure I understand the whole question here :)
What does this correspond to in the actual strings.xml file? Are we talking about strings that have already been localized?

Also, please note that Betty let me know that we should be string frozen for both a-c and Fenix since June 12th, so I believe anything new needs to wait post Fenix launch.

Reply:
Oh, sorry, here is the context:

This is about the strings on the error pages:
https://github.com/mozilla-mobile/android-components/blob/master/components/browser/errorpages/src/main/res/values/strings.xml#L69

Some of them contain HTML and we would like to change that (`<ul>` to `<ul role="presentation">`). This patch tries to do this in code by modifying the translated strings at runtime. That works, but is a bit hacky.

Creating new strings is probably not right either and that problem may come up again if we change the HTML again. So I wonder if there are other options, for example breaking those large strings with HTML up into multiple strings without HTML that we can then layout as we like...? 🤔

Reply:
As long as we're not losing all the translations on our side, I don't really have much of a preference on how to do this tbh 😶